metrics-facade-dropwizard/src/main/java/com/ringcentral/platform/metrics/dropwizard/samples/DropwizardMetricRegistryDefaultInstanceSamplesProvider.java

```diff
@@ -0,0 +1,213 @@
+package com.ringcentral.platform.metrics.dropwizard.samples;
+
+import com.codahale.metrics.MetricRegistry;
+import com.ringcentral.platform.metrics.names.MetricName;
+import com.ringcentral.platform.metrics.samples.DefaultInstanceSample;
+import com.ringcentral.platform.metrics.samples.DefaultSample;
+import com.ringcentral.platform.metrics.samples.InstanceSamplesProvider;
+import org.slf4j.Logger;
+
+import java.util.LinkedHashSet;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Supplier;
+
+import static com.ringcentral.platform.metrics.samples.SampleTypes.DELTA;
+import static com.ringcentral.platform.metrics.samples.SampleTypes.INSTANT;
+import static org.slf4j.LoggerFactory.getLogger;
+
+public class DropwizardMetricRegistryDefaultInstanceSamplesProvider implements InstanceSamplesProvider<
+        DefaultSample,
+        DefaultInstanceSample> {
+
+    private static final Logger logger = getLogger(DropwizardMetricRegistryDefaultInstanceSamplesProvider.class);
+    private static final double TIMER_FACTOR = 1.0D / (double)TimeUnit.SECONDS.toNanos(1L);
```

Comment:
MILLISECONDS is a default duration unit in Dropwizard.
For example, there is a default unit specified in `com.codahale.metrics.jmx.JmxReporter`:63: 

```java 
this.durationUnit = TimeUnit.MILLISECONDS;
```

Reply:
Do you mean that `TimeUnit.SECONDS` should be replaced with `TimeUnit.MILLISECONDS` to export millis instead of seconds?

TimeUnit.SECONDS
1)
```
MetricRegistry registry = new MetricRegistry();

Timer t1 = registry.timer("t1");
t1.update(Duration.ofMillis(50));
DropwizardMetricRegistryDefaultInstanceSamplesProvider provider = new DropwizardMetricRegistryDefaultInstanceSamplesProvider(registry);

TelegrafMetricsJsonExporter telegrafMetricsJsonExporter = new TelegrafMetricsJsonExporter(true, provider);
MetricsJson metrics = telegrafMetricsJsonExporter.exportMetrics();
System.out.println(metrics);
```
Output:
```
GroupedTelegrafMetricsJson{metricJsons={delta={t1.count=1}, instant={t1.mean=0.05, t1.max=0.05, t1.min=0.05, t1.median=0.05, t1.std_dev=0.0, t1.75_percentile=0.05, t1.95_percentile=0.05, t1.98_percentile=0.05, t1.99_percentile=0.05, t1.999_percentile=0.05}}}
```
2)
```
...
t1.update(Duration.ofMinutes(10));
...
```
Output:
```
GroupedTelegrafMetricsJson{metricJsons={delta={t1.count=1}, instant={t1.mean=600.0, t1.max=600.0, t1.min=600.0, t1.median=600.0, t1.std_dev=0.0, t1.75_percentile=600.0, t1.95_percentile=600.0, t1.98_percentile=600.0, t1.99_percentile=600.0, t1.999_percentile=600.0}}}
```

TimeUnit.MILLISECONDS
1) 
```
...
t1.update(Duration.ofMillis(50));
...
```
Output:
```
GroupedTelegrafMetricsJson{metricJsons={delta={t1.count=1}, instant={t1.mean=50.0, t1.max=50.0, t1.min=50.0, t1.median=50.0, t1.std_dev=0.0, t1.75_percentile=50.0, t1.95_percentile=50.0, t1.98_percentile=50.0, t1.99_percentile=50.0, t1.999_percentile=50.0}}}
```
2)
```
...
t1.update(Duration.ofMinutes(10));
...
```
Output:
```
GroupedTelegrafMetricsJson{metricJsons={delta={t1.count=1}, instant={t1.mean=600000.0, t1.max=600000.0, t1.min=600000.0, t1.median=600000.0, t1.std_dev=0.0, t1.75_percentile=600000.0, t1.95_percentile=600000.0, t1.98_percentile=600000.0, t1.99_percentile=600000.0, t1.999_percentile=600000.0}}}
```

Reply:
Yes, I do. 
The MILLISECONDS unit is used everywhere in Dropwizard as a default unit for time, and I believe a user as a rule expects no changes in units after migrating to our library and using Dropwizard*Provider.

Reply:
fixed, thank you.